core/types/gen_transaction_conditional_json.go

```diff
@@ -39,7 +40,11 @@ func (t *TransactionConditional) UnmarshalJSON(input []byte) error {
 		TimestampMax   *math.HexOrDecimal64  `json:"timestampMax,omitempty"`
 	}
 	var dec TransactionConditional
-	if err := json.Unmarshal(input, &dec); err != nil {
+	// --- Not Generated. Disallow unknown fields
```

Comment:
Why is this diff included? Its not related to this feature and also we should not be manually modifying autogenerated files

Reply:
Seems to be related to the go version on my Mac. It's not intended, will remove this.